modules/administration-guide/partials/con_architecture-overview-with-che-server.adoc

```diff
@@ -0,0 +1,25 @@
+[id="{prod-id-short}-architecture-overview-with-che-server_{context}"]
+= {prod-short} architecture with {prod-id-short}-server
+
+{prod-short} server is the default workspace engine.
+
+.High-level {prod-short} architecture with the {prod-short} server engine
+image::administration-guide:architecture/{project-context}-architecture-with-che-server-engine.png[]
+
+{prod} components are:
+
+Workspace controller::
+
+An always-running service managing user workspaces with the {orch-name} API.
+
+User workspaces:: 
+
+Container-based IDEs.
+
+
+After installation, the workspace controller is the only running component. {prod-short} workspaces are running on user requests.
```

Comment:
I'm not clear here how these two sentences are related. Does the workspace controller component enable the Che workspaces to be created? In other words, do these two sentences describe a cause and effect relationship? If so consider revising to make that relationship clear.

Reply:
It's a reply to the recurrent question: "I just installed Che, I can see the server components, but I don't see any of the workspaces component. Is something wrong with my installation? No, you need to start a workspace to see workspace components appear". One day we'll find a way to express that in a non Q and A manner.